### PR TITLE
Spotinst - update elastigroup integration parameters

### PR DESCRIPTION
**Pull request by jeffnoehren**
##### SUMMARY
Updating elastigroup resource parameters

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
Spotinst
##### ADDITIONAL INFORMATION
```paste below
ok: [localhost] => {
    "result": {
        "changed": true, 
        "failed": false, 
        "group_id": "sig-1e97d16d", 
        "instances": [], 
        "message": "Created group successfully."
    }
}
```

### CODE REVIEW

File: lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 DOCUMENTATION = """
 ---
 module: spotinst_aws_elastigroup
-version_added: 2.5
+version_added: 2.8
```
Comment (mattclay) [inline lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py]: The version_added on an existing module should not be changed.

**Review by mattclay — DISMISSED** [review]

File: lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py (hunk 1 above)
Comment (jeffnoehren) [reply lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py]: ok just changed it back, sorry about that!!

**Comment by talzur** [comment]
shipit

**Comment by kmcgrath** [comment]
@mattclay Is there anything else you need from our side to get this task in the release flow?

File: test/units/modules/cloud/spotinst/test_spotinst_aws_elastigroup.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import unittest
+import sys
+from mock import MagicMock
+sys.modules['spotinst_sdk'] = MagicMock()
```
Comment (mattclay) [inline test/units/modules/cloud/spotinst/test_spotinst_aws_elastigroup.py:4]: Use a pytest fixture to modify sys.modules so this doesn't pollute other unit tests.

For an example of how to do this, take a look at https://github.com/ansible/ansible/pull/52012

**Pull request closed without merging**